src/engine/enginemaster.h

```diff
@@ -252,6 +244,13 @@ class EngineMaster : public QObject, public AudioSource {
                              sizeof(long double)];
     };
 
+    std::atomic<MacroState>* m_pMacroState;
+    Macro* m_pMacroRecording;
+
+  public slots:
+    void slotStartMacroRecording(Macro* pMacro);
+    void slotStopMacroRecording();
```

Comment:
Hmm, should we be using Qt signals and slots for this? These could simply be public methods. @uklotzde what do you think?

Reply:
Hm true. I had hoped signals & slots would me decouple, but unfortunately not.

Reply:
Architecture has been changed, so this is obsolete :)